t/32-openqa_client.t

```diff
@@ -133,11 +135,11 @@ subtest 'upload other assets' => sub {
             ok(-d $chunkdir, 'Chunk directory exists');
         });
 
-    eval {
+    lives_ok {
         $t->ua->upload->asset(
             99963 => {chunk_size => $chunk_size, file => $filename, name => 'hdd_image3.xml', asset => 'other'});
-    };
-    ok !$@, 'No upload errors' or die explain $@;
+    }
+    'No upload errors';
```

Comment:
Why did you remove the die here?

Reply:
https://github.com/os-autoinst/openQA/pull/6274